Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -87,6 +94,8 @@ public void CommitBlock(long height, BlockHash hash)
                     hash,
                     NodeId);
 
+                // TODO: There's no coping mechanism when the proposed block is not present
+                // in commit stage.
```

Comment:
```suggestion
                // TODO: There's no copying mechanism when the proposed block is not present
                // in the commit stage.
```

Reply:
It was meant to say coping, however, the words seem too broad and vague. I've replaced the sentence with better context. How about this?

https://github.com/planetarium/libplanet/blob/5c60fd2283435b83f34ff871847ac9985ad4e0f6/Libplanet.Net/Consensus/ConsensusContext.cs#L97-L98

Reply:
Sorry my bad. `Coping` is the right word ;)